### PR TITLE
Rerun by redirecting to orderportal

### LINKED ISSUE #2401: Send users to the order portal when they click "Request rerun"
Re-runs can be ordered in two ways:
1) When customers are viewing a case on Scout they can click "Request rerun"
2) Via the orderportal

I suggest we replace the "Request rerun"-button on Scout with a link that send users to the order portal where they can fill in their order for a re-run. This update will save production a lot of time since re-runs ordered via the order portal starts automatically, while the ones originating from Scout has to be started manually. Additionally orders placed via order portal are more likely to be correct from start. Re-runs requested from scout are often started with incomplete information, leading to that the analysis fail, which in turn cost production a lot of time.

Before implementing this change there are a few things we need to do:
- Discuss with key customers what they think, this change will make placing an order for re-runs a bit more time consuming for them
- Come up with good argument to tell the customers when we do this change. For example we should inform the customers that this change is done so they will get their analysis faster, since request via scout leads to analyses that often fail, while re-runs ordered via the order portal are more likely to finish without problems
- Consider how to handle the re-run orders for customers that do not have access to the order portal. Are there such customers (Lund?)? Can we give them access to the order portal?

### PR DESCRIPTION
**Pull request by northwestwitch**
- Do not send a ticket by redirect user to orderportal (fix #2401)
- Since it's not automatic any more, allow users to edit rerun status of a case by clicking on new button

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN, CR

### CODE REVIEW

File: docs/admin-guide/server.md [hunk 1]
```diff
@@ -29,6 +31,15 @@ MAIL_USE_SSL = False
 # If null no badge is displayed in scout
 ACCREDITATION_BADGE = "swedac-1926-iso17025.png"
 
+# LDAP login Settings
```
Comment (northwestwitch) [inline docs/admin-guide/server.md:34]: There were a bunch on missing params here, so I've just copy-pasted the current confg file

**Comment by northwestwitch** [comment]
Ready for review now!

File: scout/adapter/mongo/case_events.py [hunk 2]
```diff
@@ -388,10 +388,13 @@ def request_rerun(self, institute, case, user, link):
         Return:
             updated_case
         """
-        if case.get("rerun_requested"):
-            raise ValueError("rerun already pending")
+        rerun_status = True
```
Comment (dnil) [inline scout/adapter/mongo/case_events.py:391]: Renaming this var could make it an easier read. The toggle part next is a bit dense on first read, especially with the function name still being request_rerun. Perhaps the function should also simply be renamed `update_rerun_status`, like the endpoint? With a new name and docstring the var name is more obvious.

File: scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html [hunk 3]
```diff
@@ -304,17 +304,28 @@
...
+  <form action="{{ url_for('cases.update_rerun_status', institute_id=institute._id, case_name=case.display_name) }}">
+    <button class="btn btn-info btn-sm form-control" type="submit" name="rerun_update">Reset rerun status</button>
+  </form>
 {% else %}
   <div id='submenu3' class="collapse">
     {% if 'RERUN_MONITOR' in config and config.RERUN_MONITOR == True %}
       {{ rerun_monitor_interact(institute, case) }}
     {% endif %}
-    <button type="button" class="btn btn-warning btn-sm form-control" data-bs-toggle="modal" data-bs-target="#rerun-modal">Request rerun</button>
+    {% if "RERUN_URL" in config %}
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html:315]: 👍

**Review by dnil — APPROVED** [review]
I am not a big fan of dropping this before an integration with rerunning cases is in place, but this has been long requested and so far no vocal protests so I guess we go with it.

The code works of course, so approved! The `request_rerun` adapter function changed its usage a bit, so at least a docstring update is needed, but even better change the name as for the endpoint!

File: scout/adapter/mongo/case_events.py (hunk 2 above)
Comment (northwestwitch) [reply scout/adapter/mongo/case_events.py:391]: I agree. Let's modify the function name, since the function purpose is different now!

Comment (northwestwitch) [reply scout/adapter/mongo/case_events.py]: Renamed also variable to is_rerun

**Pull request merged**